mxabierto/guia
# New template

## Description

- [x] New template for document *Oficio de Solicitud de Inventario de Datos*
- [x] Update document for *¿Cómo llenar el Inventario de Datos?*

## How to Test

Already available for [preview](http://carlosmaya.github.io/guia/manuales-plantillas/)

1. Go to 'Manuales y Plantillas'
2. Check for the following
![validate01- 45](https://cloud.githubusercontent.com/assets/2633527/8940550/3ab2dd48-3530-11e5-9f4f-f82a6bed9fc4.png)

3. And also
![validate02- 45](https://cloud.githubusercontent.com/assets/2633527/8940565/492166ec-3530-11e5-997d-7120f39f68e0.png)
